### ISSUE TITLE
Aggiornamento pagina ABOUT

### ISSUE THREAD

**Issue by chiccap** [emergenzeHack/covid19italia#563]
Ciao!

Mi sembra dovremmo aggiornare la pagina [about](https://www.covid19italia.info/about/).
In generale rendere le informazioni al suo interno più facilmente fruibili:
- aggiornare chi siamo (si sono aggiunte persone nuove!)
- aggiornare partners (manca slownews, cnoas, unhcr e arci...) - senza scrivere tutte quelle righe per ciascuno potremmo distinguere partners da contributors ma comunque dargli risalto almeno con il loghino!
- dare maggiore risalto alle esperienze di riuso europee - messe così si perdono e invece sono motivo di vanto!

Questo anche in ottica di quanto diceva @cristigalas sulla Fase 2 #560 

Che ne pensate?
@cristigalas @favoeva @Albertopi78 @luciaroma

**Comment by luciaroma** [comment]
Sì sono molto d'accordo, penso sia un valore aggiunto per il progetto dare spazio, anche tramite il logo, a tutte le collaborazioni di vario tipo che abbiamo in essere (partenariato, riuso, uso, integrazione etc.).

Potremmo pensare, per renderlo più chiaro e visibile, di "isolare" CHI SIAMO, spostandolo da sotto COS'E' ED ABOUT, nel menù in alto. Su Chi Siamo mettiamo sia i singoli sia tutte le associazioni ed enti che man mano e vario titolo compongono e danno valore al progetto stesso.

**Comment by cristigalas** [comment]
@chiccap @luciaroma concordo sulla necessità di aggiornare e sistemare meglio le info per renderle più visibili e chiare.

**Comment by avivace** [comment]
Sono d'accordo! Ho iniziato ad aggiungere i partner che hai suggerito, @chiccap .
Avete qualche idea su come modificare il paragrafo "Network"?

**Comment by michelerestuccia** [comment]
La frase di seguito 
"Fino ad ora, abbiamo gestito 1578 segnalazioni, accettandone e verificandone 1386 , così distribuite:"
Potrebbe essere meno ambigua rispetto al periodo di tempo, se formulata come di seguito:
" Da (?) febbraio a (?) 6 maggio 2020  abbiamo gestito 1578 segnalazioni, accettandone e verificandone 1386 , così distribuite:"

**Comment by luciaroma** [comment]
@avivace 
propongo intanto di dividere COS'E' e CHI SIAMO, in due pulsanti distinti. 

Dentro **CHI SIAMO** "una community di volontari e volontarie" e ci mettiamo
1. il team come ora e 
2. il network. 
La parte network potrebbe essere articolata: 
- con il sostegno di AA e logo; 
- in collaborazione con: tutti i loghi delle realtà che man mano si aggiungono (no elenco ma se possibile allineati, tipo in una griglia) senza descrizione e link a siti
3. contatti

Dentro COS'E' e pagina ABOUT restano

1. la descrizione del progetto
2. il riuso (e metterei qui il link al network europeo)
3. statistiche. Su queste ultime sarebbe utile averle in versione e visualizzazione dinamica. Intanto concordo con @michelerestuccia. Noi abbiamo un file dove teniamo un monitoraggio settimanale dei dati.

**Comment by michelerestuccia** [comment]
Vado di seguito in questa issue per l'aggiornamento della [versione in Inglese](https://www.covid19italia.help/en/about/).
L'ho usata per inviti a Festival Partecipazione e ho notato alcuni errori e dati vecchi

**Comment by avivace** [comment]
Ciao @michelerestuccia ! Per piccole modifiche quali typo e cambiamenti di frasi, puoi direttamente proporre delle modifiche usando l'icona della "matita" qui su GitHub, sul file che genera quella pagina: https://github.com/emergenzeHack/covid19italia/blob/master/about.md

**Comment by michelerestuccia** [comment]
Utilissima questa info, grazie Antonio, la userò più tardi. yo
